media.css

```diff
@@ -0,0 +1,125 @@
+@media (max-width: 1550px) {
+  /*.other {
+    display: flex;
+    flex-direction: column;
+  }*/
+
+  
+  .center__v {
+    padding-top: 100px;
+  }
+}
+
+  
+@media (max-width: 1453px) {
+  .middle__title {
+    right: 0px;
+    top:150px;
+  }
+
+  .content {
+    left: 0;
+    bottom: 0;
+    padding-left: 500px;
+    padding-right: 500px;
+  }
+
+  .container {
+  max-width: 100%;
+  bottom: 0px;
+  top: 900px;
+  }
+  .underside {
+  bottom: 1000px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)